ARMTemplates/101-dtl-create-vm-username-pwd-galleryimage/azuredeploy.json

```diff
@@ -1,104 +1,99 @@
 {
-    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
-    "contentVersion": "1.0.0.0",
-    "parameters": {
-        "newVMName": {
-            "type": "string",
-            "metadata": {
-                "description": "The name of the new vm to be created."
-            }
-        },
-        "existingLabName": {
-            "type": "string",
-            "metadata": {
-                "description": "The name of an existing lab where the new vm will be created."
-            }
-        },
-        "offer": {
-            "type": "string",
-            "metadata": {
-                "description": "Offer of the gallery image"
-            }
-        },
-        "publisher": {
-            "type": "string",
-            "metadata": {
-                "description": "Publisher of the gallery image"
-            }
-        },
-        "sku": {
-            "type": "string",
-            "metadata": {
-                "description": "Sku of the gallery image"
-            }
-        },
-        "osType": {
-            "type": "string",
-            "metadata": {
-                "description": "OsType of the gallery image"
-            }
-        },
-        "version": {
-            "type": "string",
-            "defaultValue": "latest",
-            "metadata": {
-                "description": "Version of the gallery image"
-            }
-        },
-        "newVMSize": {
-            "type": "string",
-            "metadata": {
-                "description": "The size of the new vm to be created."
-            }
-        },
-        "userName": {
-            "type": "string",
-            "metadata": {
-                "description": "The username for the local account that will be created on the new vm."
-            }
-        },
-        "password": {
-            "type": "securestring",
-            "metadata": {
-                "description": "The password for the local account that will be created on the new vm."
-            }
-        }
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "newVMName": {
+      "type": "string",
+      "metadata": {
+        "description": "The name of the new vm to be created."
+      }
     },
-    "variables": {
-        "resourceName": "[concat(parameters('existingLabName'), '/', parameters('newVMName'))]",
-        "resourceType": "Microsoft.DevTestLab/labs/environments"
+    "existingLabName": {
+      "type": "string",
+      "metadata": {
```

Comment:
Looks like something is wrong with this file. Was the metadata section wrongly placed here?